devfiles/java-springboot/devfile.yaml

```diff
@@ -9,44 +9,26 @@ starterProjects:
 components:
   - container:
       name: tools
-      image: maysunfaisal/springbootbuild
+      image: quay.io/eclipse/che-java11-maven:nightly
       memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       mountSources: true
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - container:
-      name: runtime
-      image: maysunfaisal/springbootruntime
-      memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       endpoints:
-        - name: '8080/tcp'
-          targetPort: 8080
-      mountSources: false
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - volume:
-      name: springbootpvc
-      size: 4Gi
+      - name: '8080-tcp'
+        targetPort: 8080
 commands:
   - exec:
       id: devBuild
       component: tools
-      commandLine: "/artifacts/bin/build-container-full.sh"
+      commandLine: mvn install
       workingDir: /projects
       group:
         kind: build
         isDefault: true
   - exec:
       id: devRun
-      component: runtime
-      commandLine: "/artifacts/bin/start-server.sh"
-      workingDir: /
+      component: tools
+      commandLine: "java -jar target/*.jar"
```

Comment:
IMHO, we should use spring-boot maven plugin for the spring boot devfile... java -jar target/*.jar is a generic Java command which aims to run a fat jar.... which can be generated my maven-assembly-plugin and maven-shade-plugin as well, not just spring-boot. 

If we're calling it the spring-boot devfile , then we should we should spring-boot specific tools and actions.

Reply:
https://www.baeldung.com/spring-boot-run-maven-vs-executable-jar

Reply:
And ofcourse, if use the spring-boot maven plugin then we need to guarantee that maven is installed inside the app container. We're using a maven command for the build command part in the devfile, so I think its safe to assume that guarantee is fulfilled.